app/views/posts/show.html.erb

```diff
@@ -22,6 +22,12 @@
     <%= raw @post.body_html %>
   </div>
 
+  <div class="tags">
```

Comment:
I see that this HTML is the same. That would be a good opportunity to use a partial. http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials